scripts/install_third_party.sh

```diff
@@ -94,9 +94,14 @@ if [ ! "$NO_SKULPT" -a ! -d "$THIRD_PARTY_DIR/static/skulpt-0.10.0" ]; then
 
     # The Skulpt setup function needs to be tweaked. It fails without certain
     # third party commands. These are only used for unit tests and generating
-    # documentation and are not necessary when building Skulpt.
+    # documentation and are not necessary when building Skulpt. Some machines
+    # also have problems running the dist tests, so the second two lines
```

Comment:
FYI not sure what "second two lines" means -- lines 2-3, or lines 3-4? Might want to clarify.

Reply:
Done.